fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -815,6 +815,11 @@ inline bool shouldBackup(MutationRef const& m) {
 std::set<Tag> CommitBatchContext::getWrittenTagsPreResolution() {
 	std::set<Tag> transactionTags;
 	std::vector<Tag> cacheVector = { cacheTag };
+	// version vector is an experimental feature that does not support encryption at rest or backup.
+	// This is indicated by returning an empty set of tags from this function.
+	if (pProxyCommitData->encryptMode.isEncryptionEnabled()) {
```

Comment:
Does it mean that the version vector won't get updated with the tags of these mutations or the mutations get sent to all logs?

Reply:
It ends up being the same behavior at a private transaction. The mutations get sent to all logs, and every tlog is included in the tpcv data structure.